scalafix/rules/src/main/scala/rsc/rules/RscCompat.scala

```diff
@@ -115,18 +115,21 @@ case class RscCompat(legacyIndex: SemanticdbIndex, config: RscCompatConfig)
       }
       env
     }
-    loop(Env(Nil), ctx.tree)
+    loop(Env(Nil), doc.tree)
     buf.result
   }
 
-  private def ascribeReturnType(ctx: RuleCtx, target: RewriteTarget): Patch = {
+  def ascribeReturnType(doc: SemanticDocument, target: RewriteTarget): Patch = {
+    implicit val impDoc = doc
     try {
       val returnTypeString = {
-        val symbol = target.name.symbol.get.syntax
+        val symbol = target.name.symbol.value
         config.hardcoded.get(symbol) match {
           case Some(returnTypeString) =>
             returnTypeString
-          case _ =>
+          case _ if index.symbols.get(symbol).isDefined =>
+            //FIXME, ignore symbols that are not found as pattern syms are returning as localn
+            //with no symbo on lookup due to https://github.com/scalameta/scalameta/issues/1725
```

Comment:
In Rsc master, FIXME comments have a special format: `// FIXME: <link to a ticket>` and TODO comments are prohibited. This: 1) makes sure that issues are only tracked in the issue tracker, 2) makes code more concise since most of the time a naked link to a ticket is sufficient to convey meaning.

Please change the new FIXME comments to accommodate this convention (todo items can be migrated to issues at https://github.com/ShaneDelmore/rsc). If you'd like, feel free to keep the existing TODO comments unchanged for the time being to simplify this pull request.